.circleci/config.yml

```diff
@@ -22,7 +45,7 @@ jobs:
     - run:
         name: Install CI tools
         command: |
-          wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+          wget -q https://www.dropbox.com/s/b52zsxgfbr1qhc9/architect?dl=1
```

Comment:
temporary way for me to test the new architect binary.
TODO: change it back to the normal way of downloading the latest release once the --group flag is tested.